tests/queries/0_stateless/02879_use_structure_from_insertion_table_with_defaults.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+CURDIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+# shellcheck source=../shell_config.sh
+. "$CURDIR"/../shell_config.sh
+
+$CLICKHOUSE_LOCAL -q "select 42 as x format Native" > $CLICKHOUSE_TEST_UNIQUE_NAME.native
+$CLICKHOUSE_LOCAL -n -q "
+create table test (x UInt64, y UInt64 default 42) engine=Memory;
```

Comment:
42 is a nice number but I would change the default here so you are sure that y wasn't filled with value from file.

Reply:
Sure, good idea